pegasus/sites.v3/code.org/views/census_map.haml

```diff
@@ -1,7 +1,15 @@
 %link{rel: "stylesheet", type: "text/css", href: "/css/census_map.css"}
 -max_height_percentage = 0.65 if max_height_percentage.nil?
 
-#gmap.full-width
+#map
+  #gmap.full-width
+  #map-footer
+    %right{style: 'float: right;'}
+      %span{style: "font-family: 'Gotham 7r', sans-serif; font-size: 20px; margin: 0 25px 0 25px;"} in partnership with
```

Comment:
It seems odd to be lowercase "in" with no obvious sentence fragment leading into it.  Should we make it "In"?

Reply:
I matched the capitalization that @tanyaparker has for the same text/image at the bottom of the page.  I can go either way, just think they should be consistent.

Reply:
Oh, I had lowercase because we had the CODE logo at first. Let's go with capitalized. And I'll fix mine.